src/components/ImagesCarousel.js

```diff
@@ -101,34 +108,71 @@ export const ImagesCarousel = ({ data, navigation, refreshTimeKey, aspectRatio }
     return renderItem({ item: carouselData[0] });
   }
 
+  if (isPaused) {
+    return (
+      <View>
+        {renderItem({ item: carouselData[index] })}
+        {showSliderPauseButton && pauseButton(isPaused, setIsPaused, sizeSliderPauseButton)}
+      </View>
+    );
+  }
+
   return (
-    <Carousel
-      data={carouselData}
-      renderItem={renderItem}
-      sliderWidth={dimensions.width}
-      itemWidth={itemWidth}
-      inactiveSlideScale={1}
-      autoplay
-      loop
-      autoplayDelay={0}
-      autoplayInterval={4000}
-      containerCustomStyle={styles.center}
-    />
+    <View>
+      <Carousel
+        data={carouselData}
+        renderItem={renderItem}
+        sliderWidth={dimensions.width}
+        itemWidth={itemWidth}
+        inactiveSlideScale={1}
+        autoplay
+        loop
+        autoplayDelay={0}
+        autoplayInterval={4000}
+        containerCustomStyle={styles.center}
+        onScrollIndexChanged={(index) => setIndex(index)}
+      />
+
+      {showSliderPauseButton && pauseButton(isPaused, setIsPaused, sizeSliderPauseButton)}
+    </View>
   );
 };
 
+const pauseButton = (isPaused, setIsPaused, size) => (
+  <TouchableOpacity
+    style={[
+      styles.pauseButton,
+      { borderRadius: normalize(size * 2), padding: normalize(size / 2) }
+    ]}
+    onPress={() => setIsPaused(!isPaused)}
+  >
+    {isPaused ? <Icon.Play size={normalize(size)} /> : <Icon.Pause size={normalize(size)} />}
+  </TouchableOpacity>
+);
+
 const styles = StyleSheet.create({
   center: {
     alignSelf: 'center'
   },
   imageContainer: {
     alignSelf: 'center'
+  },
+  pauseButton: {
+    alignItems: 'center',
+    alignSelf: 'center',
+    backgroundColor: colors.surface,
+    bottom: normalize(15),
+    justifyContent: 'center',
+    position: 'absolute',
+    right: normalize(15),
+    zIndex: 1
```

Comment:
the position is not possible in any case. there can be image descriptions in the bottom right corner

![Screenshot_1681735053](https://user-images.githubusercontent.com/1942953/232486501-7fceb045-cd66-4066-b96b-91dd93a5ade8.png)

Reply:
How about setting the location of the button on the main server?

https://github.com/smart-village-solutions/smart-village-app-app/commit/6117184332afa80c19596a82cb1e0cab2aa57624

|||
|--|--|
<img width="240" alt="image" src="https://user-images.githubusercontent.com/11755668/232791936-65e46542-f5af-4653-ba0b-c3b349df8664.png">|<img width="335" alt="image" src="https://user-images.githubusercontent.com/11755668/232792329-5f782011-cf51-4f62-a012-c2e0d3233fe1.png">

|||
|--|--|
<img width="246" alt="image" src="https://user-images.githubusercontent.com/11755668/232792415-fb765f87-383c-4a2b-a8e1-b3edc3d877e5.png"> | <img width="325" alt="image" src="https://user-images.githubusercontent.com/11755668/232792464-a7ed192f-dc3e-49b3-9336-59cc83b0ac6d.png">

|||
|--|--|
<img width="245" alt="image" src="https://user-images.githubusercontent.com/11755668/232792639-a6f3b439-4b7c-410b-9b7c-706d8af46d22.png">|<img width="325" alt="image" src="https://user-images.githubusercontent.com/11755668/232792692-d13fc202-88eb-4fd9-b0b5-5e7f22c24008.png">

Reply:
good idea. please show and ask julia to do it like this anf if other options would be nice

Reply:
I moved the button up a bit as we discussed last time, and the new features I added are still in there. I can remove it if you want.

<img width="276" alt="image-20230505-075257" src="https://user-images.githubusercontent.com/11755668/236420696-49fb22d1-e113-447c-91d4-3783ffea17dd.png">

Reply:
I'll add some more padding. 👍

Reply:
Is that okay?
with Copyright | without Copyright |
|--|--|
<img width="281" alt="image" src="https://user-images.githubusercontent.com/11755668/237037421-54fc479a-acaa-4124-99c3-b151a72113d5.png"> | <img width="282" alt="image" src="https://user-images.githubusercontent.com/11755668/237037202-4c374cb6-2833-432c-8e43-cd219795d6d8.png">

Reply:
too much space without copyright. this should be the way it was before.
and if there is copyright, please increase the padding. should be the same from copyright as without copyright from bottom of slider container.

Reply:
There was a problem posting comments on Github. So it was discussed via Slack.

https://github.com/smart-village-solutions/smart-village-app-app/pull/757/commits/a0ff624fcad6b4b694bf4d0f1b5cc930272683ec

Reply:
and how does that look like now...

Reply:
Sorry. 
|with Copyright|without Copyright|
|--|--|
![Simulator Screen Shot - iPhone 14 - 2023-05-09 at 15 12 51](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/91041dbe-5c2a-496c-8911-dd13410793a1) | ![Simulator Screen Shot - iPhone 14 - 2023-05-09 at 15 13 30](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/3a98881b-1181-4758-8ff8-37376452d962)